src/workerd/jsg/value.h

```diff
@@ -532,9 +536,10 @@ class StringWrapper {
       kj::Maybe<v8::Local<v8::Object>> parentObject) {
     v8::Local<v8::String> str = check(handle->ToString(context));
     v8::Isolate* isolate = context->GetIsolate();
+    auto& js = Lock::from(isolate);
     auto buf = kj::heapArray<char>(str->Utf8LengthV2(isolate) + 1);
     str->WriteUtf8V2(isolate, buf.begin(), buf.size(), v8::String::WriteFlags::kNullTerminate);
-    return DOMString(kj::mv(buf));
+    return js.accountedDOMString(kj::mv(buf));
```

Comment:
I noticed that DOMString/USVString are now accounted whenever they're unwrapped, but it looks like there won't be accounting if kj::String is used. Is that intentional / is it worth leaving a comment about?

Reply:
Intentional for now. I'm going to look at the various uses of the `kj::String` type mapping next but this PR was already big and I wanted to focus primarily on the `Headers` case